src/modules/crud.js

```diff
@@ -0,0 +1,44 @@
+let todos = [];
+
+const getTodos = () => todos;
+const getLS = () => {
+  JSON.parse(localStorage.getItem('todos')) || [];
+};
+
+const saveLS = () => {
+  localStorage.setItem('todos', JSON.stringify(todos));
```

Comment:
- [ ] storing the tasks is not working as expected you should save to local storage every time a task is added, deleted, or edited please it would be better if you do that

Reply:
Hi @youmari, thanks for the review. I already tried to add, edit, and delete the list items, and every time it was saved to local storage, it worked on my end. I'm not sure why it's not working on your end.

Reply:
@shyusu4 Sure, when I added a task and reload the page the task that I added is gone it should be there, and the same thing for deleting